### ISSUE TITLE
When ASK is given a molded block that contains newlines, it prints it twice

### ISSUE THREAD

**Issue by meijeru** [red/red#1910]
The first printing is the intended result of ask, the second printing is AFTER receipt of the ENTER.

```
red>> test: [
[    1 2 3
[    4 5 6
[    ]
== [
    1 2 3 
    4 5 6
]
red>> ask mold test
[
    1 2 3 
    4 5 6
[
    1 2 3 
    4 5 6
]
== ""
red>>
```

**Comment by meijeru** [comment]
Note also there is a `]`missing!

**Comment by dockimbel** [comment]
Will be fixed in the new console engine for 0.6.3.

**Comment by qtxie** [comment]
Fixed in the new GUI console.
